core/primitives/transaction_validity.hpp

```diff
@@ -20,7 +21,7 @@ namespace kagome::primitives {
    */
 
   /// Transaction is valid
-  struct Valid {
+  struct ValidTransaction {
```

Comment:
Please merge comments above into one and perhaps it's better to specify a commit in the link, because if it refers just to master it may spoil with time.